source/computer/macos-setup.rst

```diff
@@ -1,9 +1,202 @@
 macOS 配置指南
 ==============
 
+:本节作者: `王亮 <https://github.com/wangliang1989>`__\ （作者）、
+           `田冬冬 <https://me.seisman.info/>`__\ （作者）、
+           `姚家园 <https://github.com/core-man>`__\ （审稿）
+:最近更新日期: 2021-01-14
+
+----
```

Comment:
Add a note that we're assuming readers are using the latest macos system, at least>=macos catalina

Reply:
Actually I don't kown the system names, I only kown I am using 11.1.

Reply:
macOS Catalina => 10.15
macOS Big Sur => 11.x

Reply:
Done.